Cars/CarsServices/Controllers/AdminPanelController.cs

```diff
@@ -0,0 +1,69 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using CarsData.Models;
+using CarsRepository.Interface;
+//using Cars.Data.Repository;
+using CarsServices.ViewModels;
+using Microsoft.AspNetCore.Mvc;
+
+namespace CarsServices.Controllers
+{
+    public class AdminPanelController : Controller
+    {
+        ICarRepository _Car;
```

Comment:
name this field as _carRepository, and in a construcor change name of your parameter to carRepository